cli/src/trusted_commands.rs

```diff
@@ -540,3 +622,134 @@ fn unshield_funds(
 			.into_trusted_operation(trusted_args.direct);
 	let _ = perform_operation(cli, trusted_args, &top);
 }
+
+#[cfg(feature = "evm")]
+fn evm_create(cli: &Cli, trusted_args: &TrustedArgs, arg_from: &str, smart_contract: &str) {
+	let from = get_pair_from_str(trusted_args, arg_from);
+	let from_acc: AccountId = from.public().into();
+	println!("from ss58 is {}", from.public().to_ss58check());
+
+	let mut sender_evm_acc_slice: [u8; 20] = [0; 20];
+	sender_evm_acc_slice.copy_from_slice((<[u8; 32]>::from(from_acc.clone())).get(0..20).unwrap());
+	let sender_evm_acc: H160 = sender_evm_acc_slice.into();
+
+	let (mrenclave, shard) = get_identifiers(trusted_args);
+
+	let nonce = get_layer_two_nonce!(from, cli, trusted_args);
+
+	let sender_evm_substrate_addr =
+		HashedAddressMapping::<BlakeTwo256>::into_account_id(sender_evm_acc);
+	println!("Trying to get nonce of evm account {:?}", sender_evm_substrate_addr.to_ss58check());
+	let evm_account_nonce = get_layer_two_evm_nonce!(from, cli, trusted_args);
+
+	let top = TrustedCall::evm_create(
+		from_acc,
+		sender_evm_acc,
+		Vec::from_hex(smart_contract.to_string()).unwrap(),
+		U256::from(0),
+		967295,        // gas limit
```

Comment:
The user should be able to add this. E.g. make it configurable. But that only makes sense once we have a sensible charging system in our runtime, which we don't. But I'm not sure if it's the job of integritee to have a sensible gas system.. because that depends on the use case.

Reply:
I agree with what you said. I still think it could make sense to invest some time to understand that aspect better, but I guess we can do it later (if needed)

Reply:
Yes, agreed. But let's do that once we are above the MVP level :)